lib/cadet_web/router.ex

```diff
@@ -210,6 +210,12 @@ defmodule CadetWeb.Router do
   #   pipe_through :api
   # end
 
+  scope "/v2", CadetWeb do
```

Comment:
Don't create a separate scope is possible, since this endpoint is grouped as part of the other routes.